srcpkgs/racket/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'racket'
 pkgname=racket
-version=7.7
+version=7.9
 revision=1
 build_wrksrc=src
 build_style=gnu-configure
 configure_args="--enable-useprefix --disable-docs"
 hostmakedepends="gsfonts"
 makedepends="sqlite-devel gtk+-devel"
-depends="gtk+"
+depends="gtk+ libressl-devel"
 short_desc="Multi-paradigm programming language in the Lisp-Scheme family"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="LGPL-3.0-only"
```

Comment:
But the license field disagrees, so it should probably be updated.

Reply:
Ah, I see. That is an issue, since Racket relicensed: https://download.racket-lang.org/license.html though some things are still under the LGPL-3.0 licence. Should this now be Apache 2.0 ? (since that's the main licence?)

But I believe the two `vlicense` things pertain to some included code under other licences, and I don't understand why xlint generates the complaints it does.

Reply:
xlint looks directly at the `license=` field. If it made sense to `vlicense` those files, their license types should be listed in `license=` as well. See how other templates do it, I believe the correct way is using commas to split them

Reply:
right, it makes sense to me now. I checked the packaging manual for details.